src/schemas/files.py

```diff
@@ -0,0 +1,8 @@
+from pydantic import BaseModel
+
+class PDFFile(BaseModel):
+    file: bytes
+
+
+class PDFStatus(BaseModel):
+    status:str
```

Comment:
status should be an enum, containing 2 values SCANNED and NOT_SCANNED, also rename it to PDFType